tests/unit-tests/platforms/x11/test_display.cpp

```diff
@@ -103,7 +103,7 @@ class X11DisplayTest : public ::testing::Test
     {
         return std::make_shared<mgx::Display>(
                    mt::fake_shared(x11_resources),
-                   std::make_shared<std::string>("Mir on X"),
+                   std::string("Mir on X"),
```

Comment:
Since its introduction in C++11 we tend to use uniform initialization syntax (braces for initialization) everywhere.

But here there is no need as there is a converting constructor.

```suggestion
                   "Mir on X",
```